llvm/test/DebugInfo/Symbolize/XCOFF/xcoff-symbolize-data.ll

```diff
@@ -0,0 +1,161 @@
+;; This file shows that llvm-symbolizer can symbolize data symbols
+;; from the DWARF info on AIX. Note that AIX is big endian.
```

Comment:
Might be worth a FIXME or something mentioning that this test would ideally be assembly, why that isn't done/when it could be fixed ("once the integrated assembler supports blah, this test could/should be migrated to use assembly rather than LLVM IR")

Reply:
Done